src/services/db.ts

```diff
@@ -59,14 +59,14 @@ export type InstanceUrlDisplayMode =
 
 export const OVoteDisplayMode = {
   /**
-   * Show total score (upvotes + downvotes)
+   * Show upvotes and downvotes separately
    */
-  Total: "total",
```

Comment:
Was this a bad merge?

Reply:
No, I realized the sorting was strange, not its least verbose -> most verbose when you look top to bottom.

Reply:
I'm not sure I understand, to me it makes sense to have default be at the top in general.

Reply:
I mean that now it's in the order: no information (none) -> some information (only the total) -> all information (separate scores). The default is the middle, and you can choose go up or down a level to see less or more. I can change it if you want.

Reply:
Let's just keep it as is for now